README.md

```diff
@@ -2,19 +2,48 @@
 <a href="https://autorelease.general.dmz.palantir.tech/palantir/javapoet"><img src="https://img.shields.io/badge/Perform%20an-Autorelease-success.svg" alt="Autorelease"></a>
 </p>
 
-JavaPoet
-========
+<p align="center">
+<a href="https://search.maven.org/artifact/com.palantir.javapoet/javapoet"><img src="https://img.shields.io/maven-central/v/com.palantir.javapoet/javapoet" alt="Maven Central Version"></a>
+<a href="https://javadoc.io/doc/com.palantir.javapoet/javapoet"><img src="https://javadoc.io/badge2/com.palantir.javapoet/javapoet/javadoc.svg" alt="Javadoc Version"></a>
+</p>
 
-_This repo is forked from the excellent [square/javapoet](https://github.com/square/javapoet) project which is no longer actively maintained._
+# JavaPoet
 
 `JavaPoet` is a Java API for generating `.java` source files.
 
 Source file generation can be useful when doing things such as annotation processing or interacting
 with metadata files (e.g., database schemas, protocol formats). By generating code, you eliminate
 the need to write boilerplate while also keeping a single source of truth for the metadata.
 
+_This repo is forked from the excellent [square/javapoet](https://github.com/square/javapoet) project which is [no longer actively maintained](https://github.com/square/javapoet/discussions/866)._\
+Significant changes:
+
+- Changed package name to `com.palantir.javapoet`
+- Made implementation-specific fields `private` and instead added getter methods
+- Added support for record classes (`TypeSpec#recordBuilder`)
+- Added support for permitted subclasses of `sealed` types (`TypeSpec.Builder#addPermittedSubclass`)
```

Comment:
I fear this will fall out of date over time, but it's helpful to describe the enhancements/changes visibly in the readme, so it's worth that risk